db/dates.js

```diff
@@ -0,0 +1,11 @@
+const makeDates = function(start, end) {
+  let arr = [];
+  for (let dt = new Date(start); dt <= end; dt.setDate(dt.getDate() + 1)) {
+
+    arr.push(new Date(dt));
+  }
+  return arr;
+};
+
+var dates = makeDates(new Date("2020-09-23"), new Date("2021-09-23"));
```

Comment:
Are these hardcoded dates? Will this dynamically change or is this just for the sake of it getting an MVP right now?

Reply:
this was a test for functionality. I should probably delete file for clarity as the version I use to generate mock data uses new Date() to generate dates when seeded starting from whatever the current day is to one year from it.